shell/agents/Microsoft.Azure.Agent/ChatSession.cs

```diff
@@ -304,6 +316,7 @@ internal async Task<CopilotResponse> GetChatResponseAsync(string input, IStatusC
         catch (OperationCanceledException)
         {
             // TODO: we may need to notify azure copilot somehow about the cancellation.
+            // TODO: Decide whether to record the cancellation in Telemetry
```

Comment:
Maybe not needed for telemetry? @StevenBucher98 thoughts?

Reply:
Would this be if a user does ctrl+C while awaiting a response? If so I am not sure that is completely necessary, mainly unsure what the data we would get from this help us change about the product, it tells us that customer may be second guessing their responses but what can we do about that?

Reply:
Okay, seems it is not really necessary to log telemetry here. Will remove this.